### PR TITLE
Bug hunt: Stripe Payment Not Going Through (client)

### PR DESCRIPTION
**Pull request by pedbr**
### User Story

```
N/A
```
### Feature description
https://www.pivotaltracker.com/story/show/168122177

### Description of PR
The stripe payment was not going through and we were getting an 402 error "Payment Required". After refactoring the Payment method we got to the point where we found that we were not sending the needed params to the API to make the request to Stripe

### Screenshots
[Add screen shots, if you've made changes to the UI.]

### What did we learn?
After a lot of trouble-shooting we figured out exactly what we needed to send to the backend and how, in order to have a successful payment request to Stripe. We still need to figure out how to send the Current User authentication info to the backend through the headers on the Axios' POST request.

### CODE REVIEW

File: src/components/PaymentForm.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ class PaymentForm extends Component {
   render() {
     return (
       <Container>
-        <StripeProvider apiKey="pk_test_7rIPo7H0D768Gw8L7YdJqLAv">
+        <StripeProvider apiKey="pk_test_zxc8b5IX4U2OFSnqMAY2XN4i00tZz2JAqX">
```
Comment (oliverochman) [inline src/components/PaymentForm.js]: can you store this in the `.env` file

File: src/components/CheckoutForm.js [hunk 2]
```diff
@@ -23,11 +23,12 @@ class CheckoutForm extends Component {
...
   };
+  
 
   stripePayment = async stripeToken => {
-    try {
-      let response = await axios.post("/subscriptions", {
-        body: stripeToken.id
+    try { 
+      let response = await axios.post("/subscriptions", { 
+        email: this.props.currentUser.attributes.uid, stripeToken
```
Comment (oliverochman) [inline src/components/CheckoutForm.js]: do you need this param

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/PaymentForm.js (hunk 1 above)
Comment (oliverochman) [reply src/components/PaymentForm.js]: best practice dictates that you should put in the `.env` file in the codebase and in the environment variables on netlfy.

Comment (pedbr) [reply src/components/PaymentForm.js]: added to .env file, and we'll add it to netlify once it's merged with upstream

File: src/components/CheckoutForm.js (hunk 2 above)
Comment (pedbr) [reply src/components/CheckoutForm.js]: No, it's now removed.

**Pull request merged**